src/mirakuru/base_env.py

```diff
@@ -0,0 +1,106 @@
+# Copyright (C) 2014 by Clearcode <http://clearcode.cc>
```

Comment:
https://github.com/ClearcodeHQ/mirakuru/blob/master/src/mirakuru/base_env.py Edited it on master, we can update it the next release.